server/itx/server.go

```diff
@@ -250,6 +251,11 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 			}
 		}()
 		go func() {
+			defer func() {
+				if r := recover(); r != nil {
+					recovery.CrashLog(r, "/var/log")
```

Comment:
say underlying `grpcServer.Start()` panic, it will call `recovery.CrashLog()`, and here it is called again, would this be a problem (like duplicate info, or worse file not written correctly)?